src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-pipe-modal/rgw-multisite-sync-pipe-modal.component.html

```diff
@@ -0,0 +1,133 @@
+<cd-modal [modalRef]="activeModal">
+  <ng-container
+    i18n="form title"
+    class="modal-title">{{ action | titlecase }} Pipe</ng-container>
+
+  <ng-container class="modal-content">
+    <form
+      name="pipeForm"
+      #frm="ngForm"
+      [formGroup]="pipeForm"
+      novalidate>
```

Comment:
will take care from now on!

Reply:
Okay, marked unresolved since well... it is not resolved yet.

Reply:
I thought it will have unnecessary changes in the PRs, so it would be better to keep the current style

Reply:
I think there was a missunderstanding then. 

I wanted to say with `keep the current style` referring to use the same style we are using on the Dashboard.
With unnecessary file changes I meant on other PRs that will work with this code, since the code is not following current linting practices it will cause intances like this: https://github.com/ceph/ceph/pull/57715/files#diff-601d0819d8cbceeb03e674d6879d3c7b5566509783bf19b96abe356976271eb3
Where there are 200 line changes on the first file due to the linting differences. 

That's all. Hope it's more clear now what I meant

Reply:
oh, I misunderstood what you told before, do you want me to handle it in this PR, or can I do this in my other draft PR #58656 ?